### PR TITLE
docs: add some more helpful links to contributing

### PR DESCRIPTION
**Pull request by moldy530**

### CODE REVIEW

File: site/smart-accounts/signers/contributing.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ head:
 
 If you'd like to add your signer to this list, we welcome PRs! Here's how to do it:
 
-1. Fork this repo
-2. In `site/.vitepress/config.ts`, there is a `sidebar` property. Find the `Choosing a Signer` item and add a new entry in `items`. The `text` property of the entry is what will be visible in the sidebar and the `link` property should be `kebab-case`. This should match the file name in the next step. Place it above the `Externally Owned Account` guide. eg:
+1. Fork this [repo](https://github.com/OMGWINNING/aa-sdk-private)
```
Comment (avasisht23) [inline site/smart-accounts/signers/contributing.md:19]:
```suggestion
1. Fork this [repo](https://github.com/OMGWINNING/aa-sdk-staging)
```

Also, do external parties have access to this?

**Review by avasisht23 — APPROVED** [review]
Nits but lgtm

File: site/smart-accounts/signers/contributing.md (hunk 1 above)
Comment (moldy530) [reply site/smart-accounts/signers/contributing.md:19]: yup they do

File: site/smart-accounts/signers/contributing.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ If you'd like to add your signer to this list, we welcome PRs! Here's how to do
 }
 ```
 
-3. Add your document to `site/smart-accounts/signers/` and name it `your-signer-name.md` (the name should match the `link` property you added in the previous step)
+3. Add your document to [`site/smart-accounts/signers/`](https://github.com/OMGWINNING/aa-sdk-private/tree/main/site/smart-accounts/signers) and name it `your-signer-name.md` (the name should match the `link` property you added in the previous step)
```
[the remark this answers is not in the document]
Comment (moldy530) [reply site/smart-accounts/signers/contributing.md:41]: yup!

**Pull request merged**